### ISSUE TITLE
Issue feature: Tables incorrectly generated

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#87]
Hi @anil7863 ! Thank you for reporting this issue. It is the first time I see the system trying to generate a table, and it's definitely an issue we will investigate. I will write in the system's formatting rules (script-bundled.js, line 37) to NOT create tables, as I have not specified it. 

Comment by @anil7863 : 

>  I have a potential concern over the presentation of the content when asked for the issues.
> sometimes, it is presenting the issues as tabular form, but failed to have a clear accessible interface.

Evidence of the issue:
[senseui-chat-1768216882926.txt](https://github.com/user-attachments/files/24582220/senseui-chat-1768216882926.txt)
